visallo/visallo
# upgrade apache poi version to release 3.15

Testing Instructions: 
- Make sure structure file import works

Points of Regression: 
N/A

CHANGELOG
Changed: Apache Poi version from 3.15-beta1 to 3.15
